ocean_lib/data_provider/data_service_provider.py

```diff
@@ -46,19 +50,23 @@ class DataServiceProvider:
     provider_info = None
 
     @staticmethod
-    def get_http_client():
+    def get_http_client() -> object:
```

Comment:
Should return `Optional[Session]`

Reply:
HTTP client, along with DataServiceProvider, is one of the types affected by the mocking. Let's discuss these later today, after we settle the other comments.

Reply:
Fixed.